freqtrade/pairlist/VolumePairList.py

```diff
@@ -91,9 +91,9 @@ def _gen_pair_whitelist(self, pairlist: List[str], tickers: Dict,
 
         if self._pairlist_pos == 0:
             # If VolumePairList is the first in the list, use fresh pairlist
-            # check length so that we make sure that '/' is actually in the string
+            # check base currency equals to stake currency.
```

Comment:
```suggestion
            # Check if pair quote currency equals to the stake currency.
```
-- it checks quote curr, not base.

The name of the var should also be changes, since it's confusing now